OLCNE/scripts/provision.sh

```diff
@@ -464,79 +502,150 @@ bootstrap_olcne() {
+  msg "Create the Oracle Linux Cloud Native Environment: ${OLCNE_ENV_NAME}"
   echo_do olcnectl --api-server 127.0.0.1:8091 environment create \
-      --environment-name ${OLCNE_ENV} \
+      --environment-name "${OLCNE_ENV_NAME}" \
       --secret-manager-type file \
       --olcne-node-cert-path ${CERT_DIR}/production/node.cert \
       --olcne-ca-path ${CERT_DIR}/production/ca.cert \
       --olcne-node-key-path ${CERT_DIR}/production/node.key \
       --update-config
 
-  msg "Create the Kubernetes module for ${OLCNE_ENV} "
-  if [[ -z "${MULTI_MASTER}" ]]; then
+  msg "Create the Kubernetes module for ${OLCNE_ENV_NAME} "
+  if [[ ${MULTI_MASTER} == 0 ]]; then
     # Single master
     echo_do olcnectl module create \
-      --environment-name ${OLCNE_ENV} \
-      --module kubernetes --name ${OLCNE_CLUSTER} \
+      --environment-name "${OLCNE_ENV_NAME}" \
+      --module kubernetes --name "${OLCNE_CLUSTER_NAME}" \
       ${REGISTRY_K8S:+--container-registry $REGISTRY_K8S} \
-      --apiserver-advertise-address $(echo ${MASTERS} | sed -e "s/.* //") \
-      --master-nodes $(echo ${MASTERS} | sed -e "s/ /:8090,/g" -e "s/$/:8090/")\
-      --worker-nodes $(echo ${WORKERS} | sed -e "s/ /:8090,/g" -e "s/$/:8090/")
+      ${REGISTRY_OLCNE:+--nginx-image $REGISTRY_OLCNE/$NGINX_IMAGE} \
+      --apiserver-advertise-address "${MASTERS}" \
+      --virtual-ip 192.168.99.99 \
+      --master-nodes "${master_nodes}"\
+      --worker-nodes "${worker_nodes}"
   else
     # HA Multi-master
     echo_do olcnectl module create \
-      --environment-name ${OLCNE_ENV} \
-      --module kubernetes --name ${OLCNE_CLUSTER} \
+      --environment-name "${OLCNE_ENV_NAME}" \
+      --module kubernetes --name "${OLCNE_CLUSTER_NAME}" \
       ${REGISTRY_K8S:+--container-registry $REGISTRY_K8S} \
-      --virtual-ip 192.168.99.99 \
       ${REGISTRY_OLCNE:+--nginx-image $REGISTRY_OLCNE/$NGINX_IMAGE} \
-      --master-nodes $(echo ${MASTERS} | sed -e "s/ /:8090,/g" -e "s/$/:8090/")\
-      --worker-nodes $(echo ${WORKERS} | sed -e "s/ /:8090,/g" -e "s/$/:8090/")
+      --virtual-ip 192.168.99.99 \
+      --master-nodes "${master_nodes}"\
+      --worker-nodes "${worker_nodes}"
   fi
 
   msg "Validate all required prerequisites are met for the Kubernetes module"
   echo_do olcnectl module validate \
-    --environment-name ${OLCNE_ENV} \
-    --name ${OLCNE_CLUSTER}
+    --environment-name "${OLCNE_ENV_NAME}" \
+    --name "${OLCNE_CLUSTER_NAME}"
 
-  if [[ -n "${MULTI_MASTER}" ]]; then
+  if [[ ${MULTI_MASTER} == 1 ]]; then
     # Force the routing through eth1 during setup (Workaround for OLCNE-1028)
-    if [[ -z "${MASTER}" ]]; then
+    if [[ ${MASTER} == 0 ]]; then
       # Standalone operator
       msg "Set masters default route on eth1 via the operator node"
       gateway="192.168.99.100"
-    else
-      msg "Set masters default route on eth1 via worker1"
-      gateway=$(echo ${WORKERS} | sed -e 's/ .*//')
     fi
```

Comment:
As we force an operator node in multi-master, we don't have to check if we have an operator node or not.

The following should be enough
```
...
if [[ ${MULTI_MASTER} == 1 ]]; then
    # Force the routing through eth1 during setup (Workaround for OLCNE-1028)
    msg "Set masters default route on eth1 via the operator node"
    gateway="192.168.99.100"
    
    for node in ${MASTERS//,/ }; do
...
```